overrides/config/tombmanygraves.cfg

```diff
@@ -77,7 +77,7 @@ graveplayerinteractions {
 
 listconfigs {
     # If set to false, inventory lists will not be able to be spawned. Lists already in the game will not be affected. (Defaults to true.)
-    B:allowInventoryLists=true
+    B:allowInventoryLists=false
```

Comment:
if this is disabled, the event listener to remove them on death should also be removed. this change should happen in a separate PR.